Code Server 3.7.0 - Failed startup with .deb

When trying to build Code Server 3.7.0:
'''
internal/modules/cjs/loader.js:968
  throw err;
  ^
Error: Cannot find module '/usr/lib/code-server'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:1
2)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
'''
Successful build with Code Server 3.6.2:
'''
[2020-11-16T20:59:11.197Z] info  Wrote default config file to ~/.config/code-server/config.yaml
[2020-11-16T20:59:11.443Z] info  Using user-data-dir ~/.local/share/code-server
[2020-11-16T20:59:11.450Z] info  code-server 3.6.2 9bde62fbd611a7a91c5f327fa43e0d06f1379169
[2020-11-16T20:59:11.450Z] info  Using config file ~/.config/code-server/config.yaml
[2020-11-16T20:59:11.455Z] info  HTTP server listening on http://127.0.0.1:8080
[2020-11-16T20:59:11.455Z] info      - Using password from ~/.config/code-server/config.yaml
[2020-11-16T20:59:11.455Z] info      - To disable use `--auth none`
[2020-11-16T20:59:11.455Z] info    - Not serving HTTPS
'''
-->

- Web Browser: Chrome
- Local OS: Windows
- Remote OS: Google Cloud Shell & Windows
- Remote Architecture: Unknown
- `code-server --version :3.7.0

I tested the version 3.7.0 and i found the same problem:

```sh
internal/modules/cjs/loader.js:968
  throw err;
  ^

Error: Cannot find module '/usr/lib/code-server'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

The new install path of dpkg is: `/usr/lib/code-server/bin`

--- a/README.md
+++ b/README.md
@@ -35,7 +35,8 @@ We also have an in-depth [setup and configuration](./doc/guide.md) guide.
 
 ### Alpha Program 🐣
 
-We're working on a cloud platform that makes deploying and managing code-server easier. Consider [updating to 3.7.0](https://github.com/cdr/code-server/releases/tag/v3.7.0) and running code-server with our experimental flag `--link` if you don't want to worry about
+We're working on a cloud platform that makes deploying and managing code-server easier.
+Consider updating to the latest version and running code-server with our experimental flag `--link` if you don't want to worry about
 
 - TLS
 - Authentication

--- a/ci/helm-chart/Chart.yaml
+++ b/ci/helm-chart/Chart.yaml
@@ -20,4 +20,4 @@ version: 1.0.0
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 3.7.0
+appVersion: 3.7.1

--- a/ci/helm-chart/README.md
+++ b/ci/helm-chart/README.md
@@ -1,6 +1,6 @@
 # code-server
 
-![Version: 1.0.0](https://img.shields.io/badge/Version-1.0.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 3.7.0](https://img.shields.io/badge/AppVersion-3.7.0-informational?style=flat-square)
+![Version: 1.0.0](https://img.shields.io/badge/Version-1.0.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 3.7.1](https://img.shields.io/badge/AppVersion-3.7.1-informational?style=flat-square)
 
 [code-server](https://github.com/cdr/code-server) code-server is VS Code running
 on a remote server, accessible through the browser.
@@ -72,7 +72,7 @@ and their default values.
 | hostnameOverride | string | `""` |  |
 | image.pullPolicy | string | `"Always"` |  |
 | image.repository | string | `"codercom/code-server"` |  |
-| image.tag | string | `"3.7.0"` |  |
+| image.tag | string | `"3.7.1"` |  |
 | imagePullSecrets | list | `[]` |  |
 | ingress.enabled | bool | `false` |  |
 | nameOverride | string | `""` |  |

--- a/ci/helm-chart/values.yaml
+++ b/ci/helm-chart/values.yaml
@@ -6,7 +6,7 @@ replicaCount: 1
 
 image:
   repository: codercom/code-server
-  tag: '3.7.0'
+  tag: '3.7.1'
   pullPolicy: Always
 
 imagePullSecrets: []

--- a/ci/images/centos7/Dockerfile
+++ b/ci/images/centos7/Dockerfile
@@ -27,6 +27,6 @@ ENV PATH=/usr/local/go/bin:$GOPATH/bin:$PATH
 # Install Go dependencies
 ENV GO111MODULE=on
 RUN go get mvdan.cc/sh/v3/cmd/shfmt
-RUN go get github.com/goreleaser/nfpm/cmd/nfpm
+RUN go get github.com/goreleaser/nfpm/cmd/nfpm@v1.9.0
 
 RUN curl -fsSL https://get.docker.com | sh

--- a/ci/images/debian10/Dockerfile
+++ b/ci/images/debian10/Dockerfile
@@ -43,7 +43,7 @@ ENV PATH=/usr/local/go/bin:$GOPATH/bin:$PATH
 # Install Go dependencies
 ENV GO111MODULE=on
 RUN go get mvdan.cc/sh/v3/cmd/shfmt
-RUN go get github.com/goreleaser/nfpm/cmd/nfpm
+RUN go get github.com/goreleaser/nfpm/cmd/nfpm@v1.9.0
 
 RUN VERSION="$(curl -fsSL https://storage.googleapis.com/kubernetes-release/release/stable.txt)" && \
     curl -fsSL "https://storage.googleapis.com/kubernetes-release/release/$VERSION/bin/linux/amd64/kubectl" > /usr/local/bin/kubectl \

--- a/doc/install.md
+++ b/doc/install.md
@@ -80,8 +80,8 @@ commands presented in the rest of this document.
 ## Debian, Ubuntu
 
 ```bash
-curl -fOL https://github.com/cdr/code-server/releases/download/v3.7.0/code-server_3.7.0_amd64.deb
-sudo dpkg -i code-server_3.7.0_amd64.deb
+curl -fOL https://github.com/cdr/code-server/releases/download/v3.7.1/code-server_3.7.1_amd64.deb
+sudo dpkg -i code-server_3.7.1_amd64.deb
 sudo systemctl enable --now code-server@$USER
 # Now visit http://127.0.0.1:8080. Your password is in ~/.config/code-server/config.yaml
 ```
@@ -89,8 +89,8 @@ sudo systemctl enable --now code-server@$USER
 ## Fedora, CentOS, RHEL, SUSE
 
 ```bash
-curl -fOL https://github.com/cdr/code-server/releases/download/v3.7.0/code-server-3.7.0-amd64.rpm
-sudo rpm -i code-server-3.7.0-amd64.rpm
+curl -fOL https://github.com/cdr/code-server/releases/download/v3.7.1/code-server-3.7.1-amd64.rpm
+sudo rpm -i code-server-3.7.1-amd64.rpm
 sudo systemctl enable --now code-server@$USER
 # Now visit http://127.0.0.1:8080. Your password is in ~/.config/code-server/config.yaml
 ```
@@ -159,10 +159,10 @@ Here is an example script for installing and using a standalone `code-server` re
 
 ```bash
 mkdir -p ~/.local/lib ~/.local/bin
-curl -fL https://github.com/cdr/code-server/releases/download/v3.7.0/code-server-3.7.0-linux-amd64.tar.gz \
+curl -fL https://github.com/cdr/code-server/releases/download/v3.7.1/code-server-3.7.1-linux-amd64.tar.gz \
   | tar -C ~/.local/lib -xz
-mv ~/.local/lib/code-server-3.7.0-linux-amd64 ~/.local/lib/code-server-3.7.0
-ln -s ~/.local/lib/code-server-3.7.0/bin/code-server ~/.local/bin/code-server
+mv ~/.local/lib/code-server-3.7.1-linux-amd64 ~/.local/lib/code-server-3.7.1
+ln -s ~/.local/lib/code-server-3.7.1/bin/code-server ~/.local/bin/code-server
 PATH="~/.local/bin:$PATH"
 code-server
 # Now visit http://127.0.0.1:8080. Your password is in ~/.config/code-server/config.yaml

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "code-server",
   "license": "MIT",
-  "version": "3.7.0",
+  "version": "3.7.1",
   "description": "Run VS Code on a remote server.",
   "homepage": "https://github.com/cdr/code-server",
   "bugs": {

